transcom/mymove
# MB-2618 TIO move level service items one card

## Description

> Given that I am a TIO
 When I view the Move Management and Counseling service items
 Then they appear together on a single card
  And they can still be approved / denied separately

## Reviewer Notes

- In the code, you will see that multiple cards are displayed for the basic service items, ReviewServiceItems
- While clicking the next button, the card will scroll into view. This shows better when viewing the multiple basic service item cards

## Setup

Add any steps or code to run in this section to help others prepare to run your code:

```sh
make server_run
make office_client_run
```

To test:

1. Log in as TIO, office app
2. Navigate over to `http://officelocal:3000/moves/1/payment-requests` queue page
3. Click on any payment request
4. Click on the `next` button on the bottom right to navigate between service items
5. Each click should scroll to each card displayed and this should be the case for basic service items as well

## Code Review Verification Steps


## References

* [Jira story](https://dp3.atlassian.net/browse/MB-2618) for this change
* [Uses scrollIntoView()](https://developer.mozilla.org/en-US/docs/Web/API/Element/scrollIntoView)

## Screenshots

![Jul-23-2020 11-49-09](https://user-images.githubusercontent.com/1522549/88326239-b03dd300-ccda-11ea-8523-d01c2729ca2a.gif)
